### ISSUE TITLE
GUI : FED : Increase SOL Console columns

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#2993]
## Overview
During a previous conversation with the community there was a suggestion that we should have similar looking consoles for KVM over IP and SOL. After receiving feedback from a user that stated that this limits the length to 80 characters and can make reading the console difficult due to how it wraps the text. This feels unnatural as a terminal window occupies the full space of it's container. 

### Acceptance Criteria
- [ ] The width of the console window should be only constrained by its container

## UX Flow / Interaction Requirements
N/A

#### Full Window
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/1183515/110469568-69e98880-809f-11eb-8b77-85645d6c94c0.png">

#### In Page
![image](https://user-images.githubusercontent.com/1183515/110469579-6eae3c80-809f-11eb-8289-963c98c2b5ef.png)

## cURL Commands
N/A

## Notes
N/A

### Browser Tests (Chrome, Firefox, Safari (Mac), Edge (Windows))
- [ ] Tested responsive layout on supported viewport sizes
- [ ] Verified functionality works as expected, e.g. success and failure use cases
- [ ] Resolved any JavaScript errors thrown to the console

### Accessibility Tests
- [ ] Verified markup is semantic
- [ ] No reported issues in IBM Accessibility Checker (Free open source tool) https://www.ibm.com/able/toolkit/tools/

### Test Hooks
- [ ] Verify need for `data-test-id` hooks with test team

**Comment by amboar** [comment]
Are we looking to support just another fixed number of columns, or adding support for a configurable number of columns? I feel like the latter is the right direction but I realise that it might be more work than anyone is keen to sign up to.

**Comment by derick-montague** [comment]
> Are we looking to support just another fixed number of columns, or adding support for a configurable number of columns? I feel like the latter is the right direction but I realise that it might be more work than anyone is keen to sign up to.

The xterm node module provides a method called fit that changes the character length based on the window size. That was updated in a commit by Intel, so the 80 character limit is no longer an issue. However, we have set up width restrictions that we are removing to allow the console to use the full width of the window in the console opened in a new window (`full window` above) and the full width of the container in the main content area (`In page` above).
